src/classes/CW_BatchUpdateAudienceGroupMemberTest.cls

```diff
@@ -0,0 +1,109 @@
+@isTest
```

Comment:
Style issue: classes and triggers should use the "UpperCamelCase" notation. Suggestion: rename accordingly.